### ISSUE TITLE
Errors on app boot with latest version

### ISSUE THREAD

**Issue by lpil** [fazibear/airbrakex#31]
```
GenEvent handler Airbrakex.LoggerBackend installed in Logger terminating {:noreply, %{level: :error, metadata: []}} Last message: {:io_reply, #Reference<0.3354249149.804257793.86467>, :ok}
```

```
GenServer #PID<0.180.0> terminating {:noreply, %{level: :error, metadata: []}} Last message: {:gen_event_EXIT, Airbrakex.LoggerBackend, {:noreply, %{level: :error, metadata: []}}}
```

Exceptions seem to still get reported, though I'm unsure if any are missed.

**Comment by robdiciuccio** [comment]
I'm seeing the same issue with Elixir 1.6.4 and Phoenix 1.3.2. @lpil did you find a cause/resolution for this?

**Comment by lpil** [comment]
Nope, nothing. We've encountered a lot of problems with airbrake and this library so we're looking at alternatives at the moment.

**Comment by gsempe** [comment]
This issue should be fixed by this PR https://github.com/fazibear/airbrakex/pull/35 merged in the version `0.1.6`.
